style.css

```diff
@@ -0,0 +1,561 @@
+  background-blend-mode: multiply;
+}
+
+.title1 {
+  color: #d3d3d3;
+  text-align: center;
+  padding-top: 1rem;
+}
+
+.red-line {
+  display: inline-block;
+  border: 1px solid #a83023;
+  background-color: #a83023;
+  width: 52px;
+  border-radius: 2px;
+  margin-bottom: 2rem;
+}
+
+.first-card {
+  width: 90%;
+  margin: 20px;
+  padding: 20px 30px;
+  display: flex;
+  justify-items: flex-start;
+  align-items: center;
+  color: white;
+  background-color: #d3d3d31e;
+  font-size: 15px;
+  margin-bottom: 0;
+}
+
+.lecture {
+  color: #ec5242;
+  font-size: 20px;
+  font-weight: 500;
+  width: 30%;
+  padding-left: 5%;
+}
+
+.decription {
+  margin-bottom: 0;
+  width: 50%;
+  padding-left: 8%;
+}
+
+.join {
+  color: white;
+  font-size: 20px;
+  font-weight: 400;
+  padding: 30px 80px;
+  margin: 30px 40px;
+  background-color: #ec5242;
+}
+
+.see {
+  display: none;
+}
+
+/* feature speakers */
+
+.featured-speakers {
+  display: grid;
+  justify-items: center;
+}
+
+.title2 {
+  padding-top: 1rem;
+  color: #272a31;
+}
+
+.speaker_face {
+  width: 100px;
+  height: 100px;
+  margin-left: 15px;
+}
+
+.speaker {
+  width: 95%;
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  background-image: url(img/tt.png);
+  background-repeat: no-repeat;
+  padding-left: 10px;
+}
+
+.speaker-info {
+  width: 70%;
+}
+
+.name {
+  padding-top: 5px;
+  font-weight: bold;
+  color: #272a31;
+  font-size: 20px;
+  margin-bottom: 5px;
+}
+
+.grey-line {
+  width: 50px;
+  border-top: 2px solid grey;
+  margin-bottom: 5px;
+}
+
+.job-title {
+  color: #ec5242;
+  font-style: italic;
+  margin-bottom: 5px;
+}
+
+/* foother */
+
+.footer {
+  background-color: #272a31;
+  display: grid;
+  justify-items: center;
+}
+
+.icon-logo {
+  width: 100px;
+  height: auto;
+}
+
+.partners {
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+  grid-template-rows: 2fr 1fr;
+  justify-content: center;
+  column-gap: 30%;
+}
+
+.bottom-element {
+  display: flex;
+  align-content: center;
+  justify-content: flex-start;
+  gap: 30px;
+  padding-top: 10px;
+  width: 100%;
+  height: auto;
+  background-color: white;
+}
+
+.logo-icon {
+  padding-left: 3rem;
+  padding-top: 0.5rem;
+}
+
+.title-footer {
+  font-size: 10px;
+  padding-top: 10px;
+}
+
+/* about */
+
+.main-container-about {
+  background-image: url(img/background.webp);
+  background-size: 100vw 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+}
+
+.description-about {
+  color: #272a31;
+  background-color: white;
+  border: 2px solid #d3d3d3;
+  margin: 20px;
+  text-align: center;
+}
+
+.contact-about {
+  color: #272a31;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.email {
+  font-weight: bold;
+}
+
+.past {
+  text-align: center;
+  border-top: 1px solid #272a31;
+  padding-top: 30px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
- [ ] Kindly make sure to align the two containers for past classes correctly on the desktop version.

    - **YOUR APP**
      ![image](https://user-images.githubusercontent.com/87197412/210336598-d3ce6c3b-eebf-4e66-b78f-d7c18e87f3b4.png)

    - **PROVIDED DESIGN**
    
       ![image](https://user-images.githubusercontent.com/87197412/210336401-0c4effe5-bf5b-4be2-bbcd-4667a38dda23.png)